node/src/components/storage.rs

```diff
@@ -107,8 +107,13 @@ const OS_FLAGS: EnvironmentFlags = EnvironmentFlags::WRITE_MAP;
 /// Mac OS X exhibits performance regressions when `WRITE_MAP` is used.
 #[cfg(target_os = "macos")]
 const OS_FLAGS: EnvironmentFlags = EnvironmentFlags::empty();
+use static_assertions::const_assert;
+use std::mem;
+const _STORAGE_EVENT_SIZE: usize = mem::size_of::<Event>();
```

Comment:
Should have a newline between imports and consts.

Reply:
Reordered imports